syft/pkg/cataloger/golang/version.go

```diff
@@ -1,109 +1,27 @@
 // This code was copied from the Go std library.
 // https://github.com/golang/go/blob/master/src/cmd/go/internal/version/version.go
+
 // Copyright 2011 The Go Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// Package version implements the ``go version'' command.
 package golang
 
 import (
 	"bytes"
-	"encoding/binary"
-	// "cmd/go/internal/base"
+	"debug/buildinfo"
+	"errors"
+	"io"
+	"io/fs"
+	"os"
+	"runtime"
+	"runtime/debug"
+	"strings"
+
+	macho "github.com/anchore/go-macholibre"
```

Comment:
there is also a `macho` package in the stdlib; why alias this import? (without the alias items would be accessed via `macholibre`)